src/main/resources/templates/mail/alumniEmailAddress.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title th:text="#{email.reject.title(${license})}"></title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <link rel="shortcut icon" th:href="@{|${baseUrl}/favicon.ico|}" />
+</head>
+<body>
+<p th:text="'Dear ' + ${user.firstName} + ' ' + ${user.lastName} + ','"></p>
+
+<p th:text="'Thank you for your interest in the ' + ${license} + ' license for OncoKB.'"></p>
+
+<p>We noticed that you registered your account under an alumni email address. Unfortunately, we are unable to approve accounts registered under alumni addresses for any OncoKB license.</p>
```

Comment:
We noticed that you registered your account using an alumni email address. Unfortunately, we are unable to approve accounts registered with alumni addresses for any OncoKB license.